samples/drivers/i2c_fujitsu_fram/sample.yaml

```diff
@@ -6,3 +6,9 @@ tests:
       arduino_due
     tags: drivers
     depends_on: i2c
+    harness: console
```

Comment:
There is a major problem with this:

harness: console means that I can run this in sanitycheck and I should expect output and be able to parse it, however, this will only work if my board has the fixture connected.

So for example if I run this test on some board that is whitelisted using sanitycheck, I would get:

```
DEVICE: ***** Booting Zephyr OS zephyr-v1.13.0-1877-g9d14874db1 *****
DEVICE: Error writing to FRAM! error code (-5)
```

The missing logic here is in sanitycheck to actually check if a board has a certain fixture connected.

If we merge this as is, we will get lots of failures from tests that can run in sanitycheck --device-testing but have no fixtures connected to them.

Reply:
Oh! I was thinking sanitycheck has the capability to check if board has certain fixture. 

Well, can you please suggest, what is that we can do to get this working?

Reply:
I agree with @spoorthik  -- why is sanitycheck trying to run the TC if the fixture cannot be satisfied? when that is the case, it shall resort to only build